### ISSUE TITLE
Where are the configs for the two aerial datasets？

### ISSUE THREAD

**Issue by jitaameng** [XiaoxFeng/IENet#3]
1. In the configs/baselines directory, I only see yaml configs for three datasets, but the configs for the aerial datasets mentioned in the paper are not in it. So where are the configs of the aerial dataset? Or the configs of them are identical to the voc dataset?
![Snipaste_2023-08-30_09-22-59](https://github.com/XiaoxFeng/IENet/assets/49662200/64bff9ae-5a47-4f4a-9c7e-c7a69985603f)
2. And another, when training, Does the freezing parameter __C.VGG.FREEZE_AT = 2 in configs need to be changed, or should it remain frozen when training.
<img width="652" alt="image" src="https://github.com/XiaoxFeng/IENet/assets/49662200/d08627c2-9665-40d6-924f-4fa0793de676">

**Thanks.**

**Comment by XiaoxFeng** [comment]
For a fair comparison, the configs and backbone of IENet for aerial datasets are kept the same with RINet and the corresponding codes will be released soon. All of parameters for natural images (including __C.VGG.FREEZE_AT = 2 ) also are kept the same with OICR.

2023-08-30 09:30:31 "jitaameng" ***@***.***> 写道：

In the configs/baselines directory, I only see yaml configs for three datasets, but the configs for the aerial datasets mentioned in the paper are not in it. So where are the configs of the aerial dataset? Or the configs of them are identical to the voc dataset?

And another, when training, Does the freezing parameter __C.VGG.FREEZE_AT = 2 in configs need to be changed, or should it remain frozen when training.
**Thanks.**

**Comment by jitaameng** [comment]
Thank you very much for your answer!!!!!!
I also found that the annotations in the NWPU VHR-10 v2 dataset released by RINet were wrong. I noticed that someone also posted a similar problem in the RINet repository. What is the reason?
![Snipaste_2023-08-31_13-52-55](https://github.com/XiaoxFeng/IENet/assets/49662200/f3a10a17-ada6-4274-9614-5a9352927e54)

**Comment by XiaoxFeng** [comment]
Thank you very much for your reminder.Only image-level labels are used to learn the detection model under weakly supervised settings.  So the error of instance-level label of training set does not affect model training and we will correct the incorrect annotations as soon as possible.

2023-08-31 13:53:34 "jitaameng" ***@***.***> 写道：

Thank you very much for your answer!!!!!!
I also found that the annotations in the NWPU VHR-10 v2 dataset released by RINet were wrong. I noticed that someone also posted a similar problem in the RINet repository. What is the reason? @XiaoxFeng
